runtime/compiler/control/JitDump.cpp

```diff
@@ -483,9 +483,86 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
 
    try
       {
-      // if our compinfo is null, we are an application thread
+      // Process user defined options first. The user can specify any method signature to be recompiled at an arbitrary
+      // dump event. This is typically used for debugging purposes so we process this option first if it exists.
+      if (agent->dumpOptions != NULL)
+         {
+         char* indexOfDot = strchr(agent->dumpOptions, '.');
+         char* indexOfParen = strchr(agent->dumpOptions, '(');
+
+         if (indexOfDot != NULL && indexOfParen != NULL)
```

Comment:
Constants should be on the left side of `==` and `!=` comparisons; also missing parentheses.

Reply:
Fixed in 2b02a6f